README.md

```diff
@@ -15,10 +15,10 @@ Pinning to a specific tag of a task (rather than `main`) with renovate watching
 
 ## Supported Tool Versions
 
-- UDS CLI: 0.13.0
+- UDS CLI: 0.13.1
 - UDS Core: 0.24.0
-- K3D: 5.7.1
-- Lula: 0.4.2
+- K3D: 5.7.2
```

Comment:
Seems like this is getting updated but the actual install is still 5.6.3 - https://github.com/defenseunicorns/uds-common/pull/179/files#diff-5243a283b8b14b6692858f044bd84aa8a274a52804df8a29eca0cccb06382af5R23

Reply:
added a comment and updated